.github/workflows/github-labeler.yml

```diff
@@ -1,8 +1,7 @@
 name: Github Labeler
 
 on:
-  pull_request:
-    types: [opened, synchronize, reopened, ready_for_review]
+  - pull_request_target
```

Comment:
I think this syntax is wrong though? The change here should just be to add `_target` to line 4 and the rest should stay the same? 

<img width="486" alt="image" src="https://github.com/Flagsmith/flagsmith/assets/14089968/698274ea-ccdd-49aa-84fb-f09891f2ea58">